tests/cupyx_tests/scipy_tests/special_tests/test_digamma.py

```diff
@@ -10,7 +10,7 @@ class TestDigamma(unittest.TestCase):
 
     @testing.with_requires('scipy>=1.1.0')
     @testing.for_all_dtypes(no_complex=True)
```

Comment:
Could you remove `no_complex=True` option to test with complex inputs?

```suggestion
    @testing.for_all_dtypes()
```